Libraries/WebSocket/RCTWebSocketExecutor.m

```diff
@@ -83,6 +85,8 @@ - (void)setUp
     [self invalidate];
     return;
   }
+  _usageMetricsTimer = [NSTimer scheduledTimerWithTimeInterval:1.0f
```

Comment:
@tadeuzagallo I hear you for the use case with doing profiling - there needs to be an easy way to toggle it on and off.

I think there is something to be said for updating the values on a regular interval, it would look really odd to have the charts updating sporadically as bridge calls came in.

*Here's what I'm thinking*

- Move the timer over to JS `setInterval`
- Request the usage metrics over the Websocket on each cycle
- Device responds to the message with the CPU and memory metrics

This allows the `setInterval` to be easily cleared from Web UI and configured to other intervals.